mne/commands/tests/test_commands.py

```diff
@@ -177,4 +178,28 @@ def test_surf2bem():
     check_usage(mne_surf2bem)
 
 
+@slow_test
+@requires_freesurfer
```

Comment:
I added an `@ultra_slow_test` decorator just like the existing `@slow_test` one. Is it fine?
How about the behaviour of Travis with respect to this new marker?